### PR TITLE
Make a page about work related to Python 3 support on Beam site.

### PR DESCRIPTION
**Pull request by tvalentyn**

### CODE REVIEW

**Comment by tvalentyn** [comment]
R: @melap 
cc: @aaltay 
cc: @RobbeSneyders

**Comment by kennknowles** [comment]
I think the page can & should be made shorted and inlined into the guide. The current sections are super short just because I didn't have a lot of content to fill them with.

File: src/documentation/sdks/python3-support.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+layout: section
+title: "Apache Beam: Python 3 Support"
+section_menu: section-menu/sdks.html
+permalink: /documentation/sdks/python3-support/
+---
+# Python 3 Support
+
+## Status
+
+Work is in progress to add Python 3 support to Beam.  Current goal is to make Beam codebase compatible both with Python 2.7 and Python 3.4.
```
Comment (kennknowles) [inline src/documentation/sdks/python3-support.md]: Specifically, I think this intro sentence is great for putting right in the section of the other page.

File: src/documentation/sdks/python3-support.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The approach that we follow is documented in the [proposal](https://docs.google.com/document/d/1xDG0MWVlDKDPu_IW9gtMvxi2S9I0GB0VDTkPhjXT0nE/edit#heading=h.5vrnrgpkrk89).
```
Comment (kennknowles) [inline src/documentation/sdks/python3-support.md]: This could just be `[Proposal](...)`

File: src/documentation/sdks/python3-support.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+We are currently working on step 2.
+
+## Open issues
+
+See a [Kanban board](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=245&view=detail) for open issues.
```
Comment (kennknowles) [inline src/documentation/sdks/python3-support.md]: This is already covered

File: src/documentation/sdks/python3-support.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+We are currently working on step 2.
+
+## Open issues
+
+See a [Kanban board](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=245&view=detail) for open issues. 
+
+## Contributions are welcome
+
+If you are interested to help, you can select a subpackage to port and assign yourself the corresponding issue. Comment on the issue if you cannot assign it yourself.
```
Comment (kennknowles) [inline src/documentation/sdks/python3-support.md]: This sentence can just go after the bullets.

**Comment by tvalentyn** [comment]
@kennknowles Thanks, PTAL.

File: src/contribute/index.md [hunk 5]
```diff
@@ -174,6 +174,16 @@ across Java, Python, and Go, and every Beam runner.
  - JIRA: [sdk-go](https://issues.apache.org/jira/issues/?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20sdk-go) / [BEAM-2083](https://issues.apache.org/jira/browse/BEAM-2083) |
  - Contact: [Henning Rohde](mailto:herohde@google.com)
 
+### Python 3 Support
```
Comment (kennknowles) [inline src/contribute/index.md]: I would actually put this up higher. Many of the works in progress here are nearly abandoned and they should be more towards the bottom. At some point you or I or someone else should sort them.

File: src/contribute/index.md [hunk 6]
```diff
@@ -174,6 +174,16 @@ across Java, Python, and Go, and every Beam runner.
...
+
+Work is in progress to add Python 3 support to Beam.  Current goal is to make Beam codebase compatible both with Python 2.7 and Python 3.4.
+
+ - [Proposal](https://docs.google.com/document/d/1xDG0MWVlDKDPu_IW9gtMvxi2S9I0GB0VDTkPhjXT0nE)
+ - [Kanban Board](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=245&view=detail)
+
+Contributions are welcome! If you are interested to help, you can select a subpackage to port and assign yourself the corresponding issue. Comment on the issue if you cannot assign it yourself.
+When submitting a new PR, please tag @RobbeSneyders, @aaltay, and @tvalentyn.
```
Comment (kennknowles) [inline src/contribute/index.md]: You could make these links to e.g. https://github.com/aaltay. It just adds snazzy hypertextualism and context.

**Comment by kennknowles** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: MergeBot encountered an unexpected error while processing this PR: 503 Server Error:  for url: https://builds.apache.org/api/python?tree=jobs%5Bname%2Ccolor%2Curl%5D.

**Comment by kennknowles** [comment]
@asfgit merge

**Comment by tvalentyn** [comment]
Sent https://github.com/apache/beam-site/pull/442 for GitHub usernames. I could sort the ongoing efforts alphabetically, but if ranking is preferred, I'd suggest this to be done by a PMC or committer or who has broader context.

**Pull request merged**